### PR TITLE
test(e2e,playwright): hotkeys

### PR DESCRIPTION
**Pull request by raditotev**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Ref #51705

### CODE REVIEW

File: e2e/hotkeys.spec.ts [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    browserName === 'webkit',
+    'Failing on webkit for no apparent reason. Can not reproduce locally.'
+  );
+
+  // Enable keyboard shortcuts
+  await page.goto('/settings');
+  const keyboardShortcutsOnButton = page
+    .getByRole('button', { name: translations.buttons.on, exact: true })
+    .nth(2);
+  await keyboardShortcutsOnButton.click();
```
Comment (ojeytonwilliams) [inline e2e/hotkeys.spec.ts:26]:
```suggestion
  const keyboardShortcutGroup = page.getByRole('group', {
    name: translations.settings.labels['keyboard-shortcuts']
  });
  await keyboardShortcutGroup
    .getByRole('button', { name: translations.buttons.on, exact: true })
    .click();
```
Depending on the position in the list of 'on' buttons is a sketchy thing to rely on. If that changes, the test will fail and it won't be immediately obvious why.

The accessible name is less likely to change and, if it does, the test fails in an obvious way.

**Review by ojeytonwilliams — CHANGES_REQUESTED** [review]
Nice work, @raditotev. Just one nitpick:

File: e2e/hotkeys.spec.ts (hunk 1 above)
Comment (raditotev) [reply e2e/hotkeys.spec.ts:26]: Love it. Thanks. I'll go ahead and replace the locator in my other [PR](#52155) with this one.

**Review by huyenltnguyen — APPROVED** [review]
LGTM 👍

File: e2e/hotkeys.spec.ts (hunk 1 above)
Comment (ojeytonwilliams) [reply e2e/hotkeys.spec.ts]: Awesome, thanks!

**Review by ojeytonwilliams — APPROVED** [review]
LGTM :+1:

**Pull request merged**